openvpn-install.sh

```diff
@@ -596,8 +599,12 @@ function installOpenVPN () {
 		PASS=${PASS:-1}
 		CONTINUE=${CONTINUE:-y}
 
-		# Behind NAT, we'll default to the publicly reachable IPv4.
-		PUBLIC_IPV4=$(curl ifconfig.co)
+		# Behind NAT, we'll default to the publicly reachable IPv4/IPv6.
+		if [[ $IPV6_SUPPORT == "y" ]]; then
+			PUBLIC_IPV4=$(curl https://ifconfig.co)
+		else
+			PUBLIC_IPV4=$(curl -4 https://ifconfig.co)
+		fi
 		ENDPOINT=${ENDPOINT:-$PUBLIC_IPV4}
```

Comment:
I guess we can rename `PUBLIC_IPV4` to `PUBLIC_IP` now 🙂

Reply:
Easy, done. :)